finetune/scheduler.py

```diff
@@ -18,8 +18,14 @@ def bytes_to_meg(x):
 
 def scheduled(fn):
     @functools.wraps(fn)
-    def scheduled_predict(self, model_file, x, *args, key=None, config_overrides=None, **kwargs):
-        model = self._rotate_in_model(model_file, key=key, config_overrides=config_overrides)
+    def scheduled_predict(
+        self, model_file, x, *args, key=None, config_overrides=None, **kwargs
+    ):
+        # this is just for backwards compat, should always have a blob key going forward
+        blob_key = kwargs.get("sobj_url", None)
```

Comment:
Could we use a more generic name for this -- e.g. "cache_key"?

Reply:
Total nit, but since it's open source it feels a bit weird having super indico specific terminology like this included.  And it's a general concept -- you could key off of anything really.